### PR TITLE
Add a script to remove paste from CLI

### PR DESCRIPTION
**Pull request by Shir0kamii**

### CODE REVIEW

File: zerobin/remove_paste.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from zerobin.paste import Paste
+from clize import run
+import re
+
+paste_url = re.compile('/paste/(?P<paste_id>.*)#(?P<key>.*)')
```
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: A comment to explain the regex would be awesome.

File: zerobin/remove_paste.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+def unpack_paste(paste):
+    """Take either the ID or the URL of a paste, and return its ID"""
+
+    try_url = paste_url.search(paste)
+
+    if try_url:
+        return try_url.group('paste_id')
+    return paste
```
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: You can rewrite the three lines in one ternary expression: `return try_url.group('paste_id') if try_url else paste`

File: zerobin/remove_paste.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+def remove_paste(*paste_list, quiet:'q'=False):
```
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: pastes seems to me a better name for paste_list.
`quiet:'q'=False`, is this type hinting?

File: zerobin/remove_paste.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    """
+    Remove pastes, given its ID or its URL
+
+    paste_list: Liste of paste, given by ID or URL
```
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: A list of past.
Excuse my French!

File: zerobin/remove_paste.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    for paste_uuid in map(unpack_paste, paste_list):
+        try:
+            Paste.load(paste_uuid).delete()
+        
+        except ValueError:
+            if not quiet:
+                print('Paste {} doesn\'t exist'.format(paste_uuid))
+
+        else:
```
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: I don't feel like that the else block is useful.
You could write this after the delete in the try.

**Comment by RaitoBezarius** [comment]
Reviewed, let me know what do you think about my comments!

File: zerobin/remove_paste.py (hunk 2 above)
Comment (Shir0kamii) [inline zerobin/remove_paste.py]: I think it's clearer this way

File: zerobin/remove_paste.py (hunk 3 above)
Comment (Shir0kamii) [inline zerobin/remove_paste.py]: The annotation is not for type hinting. It's one of the way you can give more details about the parameter to clize. Here, I tell it that the short version of --quiet is -q

File: zerobin/remove_paste.py (hunk 2 above)
Comment (RaitoBezarius) [inline zerobin/remove_paste.py]: Agreed!

**Comment by RaitoBezarius** [comment]
Reviewed, ready to merge! I let you click the button!

**Pull request merged**